src/test/cypress/e2e/course/CourseManagement.cy.ts

```diff
@@ -45,7 +45,7 @@ describe('Course management', () => {
     describe('Manual student selection', () => {
         let course: Course;
 
-        beforeEach('Create course', () => {
+        before('Create course', () => {
```

Comment:
Can you also change this in line 86?

Reply:
And line 289? Maybe you can just move the login part of the `before`-s to the outermost `describe` scope 🤔

Reply:
There are also tests that create and delete courses using the UI. I think having one existing course for all of these tests plus the courses that get added or deleted by the "user" would make this file quite hard to read. While this could save a few requests, I would not implement it like this. Also the main reason why I wanted to avoid to many delete calls (20 seconds timeout) got now changed as well, so it no longer is a big issue.

Reply:
Ok! Also, as you mentioned via Slack, we now have login sessions anyway => probably not much to gain here => resolving this